docs/user/build-customization.rst

```diff
@@ -4,21 +4,21 @@ Build process customization
 Read the Docs has a :doc:`well-defined build process </builds>` that works for many projects.
 We also allow customization of builds in two ways:
 
-`Extend the build process`_
-    Keep using the default build process,
-    adding your own commands.
+Extend or partially override the build process
+   Keep using the default commands for MkDocs or Sphinx,
+   but extend or override the ones you need.
 
-`Override the build process`_
-    This option gives you *full control* over your build.
-    Read the Docs supports any tool that generates HTML.
+Completely override the build process
+   This option gives you *full control* over your build.
+   Read the Docs supports any tool that generates HTML.
 
-Extend the build process
-------------------------
+Extend or override the build process
+------------------------------------
 
-In the normal build process,
-the pre-defined jobs ``checkout``, ``system_dependencies``, ``create_environment``, ``install``, ``build`` and ``upload`` are executed.
-Read the Docs also exposes these jobs,
-which allows you to customize the build process by adding shell commands.
+In the normal build process, the pre-defined jobs ``checkout``, ``system_dependencies``,  and ``upload`` are executed.
+If you define a :ref:`config-file/v2:sphinx` or :ref:`config-file/v2:mkdocs` configuration,
+the ``create_environment``, ``install``, and ``build`` jobs will use the default commands for the selected tool,
+otherwise, they will default to run nothing.
```

Comment:
Can't do suggest for some reason :shrug: 

I'd make this two sentences.

```
...jobs will use the default commands for the selected tool.
If no tool configuration is specified, these jobs won't execute anything by default.
```

Reply:
probably because the PR was already merged :)